package.json

```diff
@@ -28,21 +29,63 @@
   "homepage": "https://github.com/octokit/app.js#readme",
   "types": "index.d.ts",
   "dependencies": {
-    "@octokit/request": "^3.0.0",
-    "@types/lru-cache": "^5.1.0",
+    "@octokit/request": "^4.1.0",
     "jsonwebtoken": "^8.3.0",
     "lru-cache": "^5.1.1"
   },
   "devDependencies": {
-    "chai": "^4.1.2",
-    "coveralls": "^3.0.2",
+    "@gr2m/pika-plugin-bundle-web": "^0.3.16-patch-issue-46-6",
```

Comment:
a new version of `@pika/plugin-bundle-web` has been released, we no longer need my fork. Can you bump all the `@pika/*` packages to `^0.4.0`